src/WebJobs.Script.WebHost/Middleware/HostAvailabilityCheckMiddleware.cs

```diff
@@ -18,13 +18,16 @@ public HostAvailabilityCheckMiddleware(RequestDelegate next, ILoggerFactory logg
             _logger = loggerFactory.CreateLogger<HostAvailabilityCheckMiddleware>();
         }
 
-        public async Task Invoke(HttpContext httpContext, WebHostResolver resolver)
+        public async Task Invoke(HttpContext httpContext)
         {
             using (Logger.VerifyingHostAvailabilityScope(_logger, httpContext.TraceIdentifier))
             {
                 Logger.InitiatingHostAvailabilityCheck(_logger);
 
-                bool hostReady = await WebScriptHostManager.DelayUntilHostReady(resolver);
+                // TODO: DI (FACAVAL) Perform host check... moving forward, this information
+                // will be provided by the host startup service.
+                //bool hostReady = await WebScriptHostManager.DelayUntilHostReady(resolver);
```

Comment:
Without this in place we can get race conditions - won't that affect tests?

Reply:
This is still in place, that's why the tests work. A lot of things have been refactored/moved, so you'll see a lot of deletions just looking at the diffs. Might be worth having the project up.